lib/api/components.php

```diff
@@ -22,15 +24,14 @@
  * @return bool Will always return true.
  */
 function beans_load_api_components( $components ) {
-
 	static $loaded = array();
 
 	$root = BEANS_API_PATH;
 
 	$common = array(
 		'html'         => array(
 			$root . 'html/functions.php',
-			$root . 'html/class.php',
+			$root . 'html/class-beans-attribute.php',
```

Comment:
Just changing the filename here without renaming the original file in the HTML API will break Beans. Why? That file doesn't exist. Right?

This change is included in PR #114.